docs/src/getting-started/checkpoints.rst

```diff
@@ -25,20 +25,30 @@ or
     mtt train options.yaml -c model.ckpt
 
 Checkpoints can also be turned into exported models using the ``export`` sub-command.
+The command requires the *architecture name* and the saved checkpoint *path* as
+positional arguments
 
 .. code-block:: bash
 
-    mtt export model.ckpt -o model.pt
+    mtt export experimental.soap_bpnn model.ckpt -o model.pt
```

Comment:
Completely unrelated, but I will open an issue to have this removed (it should be easy and I don't like it too much)

Reply:
You don't like the syntax itself or the line in the docs?

The syntax we can't remove because we need the corresponding archtecture name to load a checkpoint.

Reply:
Having the name on the command line should be avoidable by requiring an `architecture_name` field in the checkpoint (but then this rule must be enforced for all architectures and added to "how to add a new architecture")